src/ContainerCommitModal.jsx

```diff
@@ -0,0 +1,174 @@
+    }
+
+    // TODO
+    handleInputChange(event) {
+
+    }
+
+    // TODO
+    handleFormatChange(event) {
+
+    }
+
+    // TODO
+    handleCommit() {
+
+    }
+
+    handleCancel() {
+        this.props.handleCancelContainerCommitModal();
+        this.setState(this.initialState);
+    }
+
+    render() {
+        let onbuilds =
+            this.state.onbuild.map((bud, idx) => (
+                <div key={"onbuildvar" + idx} id="select-claimed-onbuildvars" className="containers-run-onbuildvarclaim containers-run-inline" >
+                    <form className="form-inline">
+                        <button type="button" className="btn btn-default fa fa-plus" onClick={this.handleAddOnbuild} />
+                        <button type="button" className="btn btn-default pficon-close" onClick={() => this.handleRemoveOnbuild(idx)} />
+                        <div className="form-group">
+                            <input type="text" name="onbuildvar_key" className="form-control" onChange={(evt) => this.handleOnbuildsInputChange(idx, evt)} />
+                        </div>
+                    </form>
+                </div>
+            ));
+
+        let commitContent =
+            <div>
+                <table className="form-table-ct">
+                    <tbody>
+                        <tr>
+                            <td align="right"><label className="control-label" translatable="yes">Container Name</label></td>
+                            <td colSpan="3">
+                                <span className="container-name" />{this.props.containerWillCommit.Name}
+                            </td>
+                        </tr>
+                        <tr>
+                            <td align="right"><label className="control-label" translatable="yes">Format</label></td>
+                            <td colSpan="3">
+                                <label htmlFor="format-oci">
+                                    <input type="radio" id="format-oci" value="oci" checked={this.state.selectedFormat === 'oci'} onChange={(event) => this.handleFormatChange(event)} />
+                                    oci
+                                </label>
+                                <label htmlFor="format-docker">
+                                    <input type="radio" id="format-docker" value="docker" checked={this.state.selectedFormat === 'docker'} onChange={(event) => this.handleFormatChange(event)} />
+                                    docker
+                                </label>
+                            </td>
+                        </tr>
+                        <tr>
+                            <td align="right"><label className="control-label" translatable="yes">Image Name</label></td>
+                            <td colSpan="3">
+                                <input name="imageName" className="form-control-commit container-imageName" type="text" onChange={this.handleInputChange} required="true" />
```

Comment:
Please leave out the input field specific class names. You don't define CSS for them, and also really shouldn't - these input fields are all alike, they shouldn't all have a different styling.